data/hai/unfettered 0 prologue.txt

```diff
@@ -506,57 +509,1298 @@ mission "Unfettered Jump Drive Trading"
+	on visit
+		dialog phrase "generic missing stopover or cargo"
+	on complete
+		log `Brought medial supplies to Darkcloak to prevent an epidemic.`
+		log "People" "Sookootow" `An Unfettered Chief worried for his people, inside Hai or Unfettered territory. He seems to believe that all Hai are equal, and as a member of the council he has significant influence.`
+		payment 1500000
+		conversation
+			`The staff seem greatly relieved when you return promptly with the medical supplies. As you're returning to your ship, one of them catches up to you. "Hello, captain <last>. I am Merki, manager of this hospital. I wanted to express my deepest thanks to you and give you the <payment> personally. When you have the time, make sure to check the job board for more jobs like this, it would mean a great deal to me."`
+			`	Once Merki leaves, you notice a recognizable face: the same Hai Chief from Firelode, speaking with someone who, judging by their demeanour and attire, could be the hospital's chief medical officer. He soon sees you as well and hurriedly finishes his conversation, seeming more than a bit emotional as he then moves to speak with you. "I knew you could be counted on as a friend to the Unfettered! With the swift response of captains like you, it seems like it has calmed down here. We did not before have the time to get properly introduced. I am Sookootow." He makes a little gesture, you assume of deference. You try to imitate the motion, apparently badly, amusing Sookootow. "Worry not, <first> <last>, I already know who you are."`
+			choice
+				`	"I only did what little I could do to help save people. Hopefully I will be able to do more next time we meet."`
+					decline
+				`	"What could have caused this crisis? Does it happen often?"`
+				`	"Are all Unfettered soldiers? I see a lot of you armed and seemingly ready for action."`
+					goto soldiers
+			`	"It depends on who you ask... Some will say the aliens, your people, are the reason, given the Hai have much contact with them, and we with the Hai. Of course, the Hai who sympathise with our cause do so with many others as well, such as helping humans with low funds relocate to their space. As you have probably noticed, we ourselves do not get many visits from outsiders; they obviously do not feel very welcomed when getting shot at for their supplies. That means we are not protected against the diseases they may bring.`
+			`	"As for the frequency, the precarious conditions here mean this happens too often for our liking. As a result, we have a habit of quickly reacting to any seemingly deadly diseases. That means that sometimes we worry for nothing, but this was not one of those times. We are a strong people, but even a strong Heshi is nothing in shallow waters. It is a matter of life or death that the hospital be in its best possible state, but it is only that way thanks to much work and investments. The many places in which our people live are not quite as sophisticated."`
```

Comment:
Just flagging this section; from a medical and disease vector standpoint this whole explanation doesn't really fit with the situation or the question.
It's an isolated section so it doesn't have any impact on anything else, but it should be re-written from scratch at some point. (I'll do it if I get time.)

Reply:
well it is propaganda that humans bring disease to Hai probably

as for the way to react they do it this way because the fact ppl have low level of health in general (food issues and crammed in buildings) means this will happen often

Reply:
I think its fine because I don't imagine the Unfettered are competent at dealing with this; at least not him, he is no specialist of disease propagations
mb he could mention that, or you could say smth like before, ~~but you're just a textile worker~~